test/pseudonymsys_ec_test.go

```diff
@@ -13,7 +13,7 @@ import (
 
 func TestPseudonymsysEC(t *testing.T) {
 	dlog := dlog.NewECDLog(dlog.P256)
-	caClient, err := client.NewPseudonymsysCAClientEC(testGrpcServerEndpoint)
+	caClient, err := client.NewPseudonymsysCAClientEC(testGrpcClientConn)
```

Comment:
I'm not sure if we're allowed to refer global variables from other test files. There is no guarantee that communication_test.go will be executed prior pseudonymsys_ec_test.go. Also, in communication_test.go we invoke
```go
testGrpcClientConn.Close()
```
so I'm not sure how is this currently even working?

On some other project*, we resolved similar situation by using gocheck framework for unit testing instead of builtin Go tests. The framework allows you to create a struct with arbitrary fields (you could store connection there) and specify functions like
```
SetUpSuite // before running all the tests
SetUpTest //before running each test
TearDownTest // after running each test
TearDownSuite // after running all the tests
```
Do you think it would make sense to start using the gocheck also for emmy?

\* https://github.com/mikelangelo-project/capstan/blob/master/cmd/package_test.go